packages/sockethub-platform-xmpp/src/index.js

```diff
@@ -313,7 +311,7 @@ class XMPP {
       const status = job.object.content || '';
       // setting presence
       this.debug('setting presence: ' + show + ' status: ' + status);
-      this.__client.send(xml(show, { type: status })).then(done);
+      this.__client.send(xml("presence", { show: show }, { status: status })).then(done);
```

Comment:
If the variable name is the same as the property name, you only need to put the name in modern JS, like so:

```suggestion
      this.__client.send(xml("presence", { show }, { status })).then(done);
```